posthog/tasks/usage_report.py

```diff
@@ -401,7 +401,7 @@ def get_teams_with_billable_event_count_in_period(
         f"""
         SELECT team_id, count({distinct_expression}) as count
         FROM events
-        WHERE timestamp between %(begin)s AND %(end)s AND event != '$feature_flag_called' AND event NOT IN ('survey sent', 'survey shown', 'survey dismissed')
+        WHERE timestamp between %(begin)s AND %(end)s AND event NOT IN ('$feature_flag_called', 'survey sent', 'survey shown', 'survey dismissed') AND NOT startsWith(event, '$$')
```

Comment:
flyby to make sure you test this in metabase for 1 day & 30 days of data, somewhat sure 30 days wouldn't work, but 1 day should be atmost 2x slower?

Reply:
Haven't tested it yet. Just started the PR but good call I'll have to see if it is at all reasonable